uwsgi/admin.wsgi

```diff
@@ -34,7 +34,7 @@ if STAGING or LOCALDEV:
     SYSTEM_ACCOUNTS.extend(["balrog-stage-ffxbld", "balrog-stage-tbirdbld"])
     DOMAIN_ALLOWLIST.update(
         {
-            "ftp.stage.mozaws.net": ("Firefox", "Fennec", "Devedition", "SeaMonkey", "Thunderbird", "Pinebuild"),
+            "ftp.stage.mozaws.net": ("Firefox", "Fennec", "Devedition", "SeaMonkey", "Thunderbird", "Pinebuild", "SystemAddons"),
```

Comment:
Sorry about the unrelated change @gbrownmozilla. This allows system addon releases in staging & locally to point to the staging archive `ftp.stage.mozaws.net`